test/distrib/bazel/test_single_bazel_version.sh

```diff
@@ -15,12 +15,26 @@
 
 set -ex
 
-if [ "$#" != "1" ] ; then
+if [ "$#" == "0" ] ; then
     echo "Must supply bazel version to be tested." >/dev/stderr
     exit 1
 fi
 
 VERSION="$1"
+shift 1
+
+# Accept the list of test shards to run on the commandline
+TEST_SHARDS=("$@")
+
+# If list of shards to run hasn't been specified on the command line,
+# run all the known shards.
+if [ "${#TEST_SHARDS[@]}" == "0" ] ; then
+  TEST_SHARDS=(
+    buildtest
+    distribtest_cpp
```

Comment:
Seems like these defaults should somehow be kept synchronized with the `TEST_DIRECTORIES` variable below.

Reply:
done.